crates/nu-command/src/generators/seq_char.rs

```diff
@@ -65,7 +65,7 @@ impl Command for SeqChar {
 }
 
 fn is_single_character(ch: &str) -> bool {
-    ch.is_ascii() && ch.len() == 1 && ch.chars().all(char::is_alphabetic)
+    ch.is_ascii() && ch.len() == 1 
```

Comment:
Should we permit the non-printable characters `0x00..0x20` and `0x7f`?

Else this could be `is_ascii_graphic`

Reply:
@sholderbach and @fdncred  What if non-printable ASCII are Required should I add a flag for printable ASCII only??
and I forgot to commit the test case will do it now

Reply:
I was thinking about non-printable characters as well. My preference would be for those to be opt-in with a flag.

Reply:
I don't think we need a flag as the non-printables are strictly before and after the interval of printables, and if you want them you will need to use the corresponding escapes. Only if you try to use `\n`/`\t` assuming they would be "normal" characters you might get a asuprise.

Reply:
Let me implement a method to convert values into either printable text or binary format, enabling users to verify a script's coverage and behavior across the entire ASCII character set.

Reply:
You know, before getting too far with this, we should ask if the `seq char` is all that useful as a built-in command.  I think what's really missing is the "inverse" of the `char` command.  If user's had a way to "look up" the codepoint value for a character, then `seq chars` could just be a Standard Library command.

Reply:
See #14264 for an alternative

Reply:
I'd vote against removing seq char. It's an easy way to generate character ranges.

Reply:
@anomius I think I'd recommend just limiting it to 32..126 (inclusive).  That is, a space should probably be the only "unprintable" character that is allowed.  I don't think I'd worry about a flag for now.

Reply:
BTW, i'm not as opposed as i originally was to the scripted solution in https://github.com/nushell/nushell/issues/14264#issuecomment-2457952238. if we did move forward with it, i'd want it to be in the global scope.

Reply:
@fdncred Can you clarify "in the global scope" in this context?

Reply:
when nushell starts `seq char` should be available to run without doing any use command.

Reply:
Ah. In that case I wouldn't recommend putting it in the Standard Library since it would be parsed every startup, of course.

Is it really that commonly used?

Reply:
The only uses I can find are examples and leetcode/AdventOfCode kind of stuff.

https://github.com/search?q=lang%3ANushell+%22seq+char%22&type=code